internal/common/baseclient/basejudge.go

```diff
@@ -0,0 +1,87 @@
+package baseclient
```

Comment:
`basejudge_test.go`? link to PR adding this test.

![image](https://user-images.githubusercontent.com/33488131/103108281-556c2700-463e-11eb-9ea9-7c8d5d2fefc3.png)

Reply:
Fair enough, could we postpone this to another PR pls, these are going to change a bit in post-mvp work.